web/views/user_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Janitor.UserView do
+  use Janitor.Web, :view
+
+  def render("me.json", %{user: user}) do
```

Comment:
What's the point of having a parameter here which will never change? Can't you just pass `user` here and be done with that?